src/components/Henvisning1963/Henvisning1963.tsx

```diff
@@ -0,0 +1,56 @@
+import { FormattedMessage } from 'react-intl'
+
+import { BodyLong, Button, Heading, HStack, VStack } from '@navikt/ds-react'
+
+import { Card } from '../common/Card'
+import { externalUrls } from '@/router'
+import { wrapLogger } from '@/utils/logging'
+
+interface IProps {
+  onAvbryt: () => void
+}
+
+const gaaTilDetaljertKalkulator = () => {
+  window.open(externalUrls.detaljertKalkulator, '_self')
+}
+
+export const Henvisning1963: React.FC<IProps> = ({ onAvbryt }) => {
+  return (
+    <Card
+      data-testId="henvisning-1963"
+      aria-live="polite"
+      hasLargePadding
+      hasMargin
+    >
+      <Heading level="2" size="medium" spacing>
```

Comment:
Jeg tror at du kan bruke vår komponent Card.Content her for å få ferdig layout med tittel, body og knapper. Du kan søke den opp og se om det passer. Gjelder den andre Henvisningkomponenten også